utils/package_checker.py

```diff
@@ -0,0 +1,27 @@
+#!/bin/env python
+import json
+import sys
+import os.path
+
+# Exit 2 if upstream package list doesnt exist
+# Exit 1 if file exists, but package isnt in the list
+
+jsonpath = 'fedora-atomic/fedora-atomic-host-base.json'
+
+# Make sure json file exists
+if not os.path.isfile(jsonpath):
+    print("Could not find upstream package json file")
+    sys.exit(2)
+
+jsonfile = open(jsonpath, 'r').read()
```

Comment:
When opening/reading files in python, it's better to use the context manager approach. This handles closing a file if an error occurs, etc. This section would be better written as:

```
with open(jsonpath, 'r') as f:
    atomicjson = json.load(f)
```